solutions/python_azure_sdk/Makefile

```diff
@@ -0,0 +1,46 @@
+TOP = $(abspath ../../)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+
+OPTS = --nobrk
+
+ifdef STRACE
+OPTS += --strace
+endif
+
+all: rootfs
+
+appdir-keyvault_identity:
+	$(APPBUILDER) -d Dockerfile -v -e '--build-arg TAG=azure-mgmt-keyvault_9.1.0 --build-arg PACKAGES=keyvault_identity/packages.txt' -o keyvault_identity/appdir
+
+appdir-storage:
+	$(APPBUILDER) -d Dockerfile -v -e '--build-arg TAG=azure-mgmt-storage_19.0.0 --build-arg PACKAGES=storage/packages.txt' -o storage/appdir
+
+appdir-storage-blob:
+	$(APPBUILDER) -d Dockerfile -v -e '--build-arg TAG=azure-storage-blob_12.8.1 --build-arg PACKAGES=storage-blob/packages.txt' -o storage-blob/appdir
+
+rootfs-keyvault_identity: appdir-keyvault_identity
+	$(MYST) mkext2 keyvault_identity/appdir keyvault_identity/rootfs
```

Comment:
The short answer is: I have tried to build&run all the packages in one rootfs but not all packages passed.

The first reason is versioning, you can see I use different tags for different packages. All packages do not pass with the same tag.

The second reason is, when I combine them, sometimes one package cannot find the definition of some functions of the same module but is defined in another package. For example, `storage` and `storage.blob` are both within storage module `storage. If I install them both, then the `storage` module is corrupted. I think this is because in development environment, `pip install -e .` is used to install packages locally. So you can setup development environment for one package but not all packages.

If you observe the python sdk pipeline, each package has individual pr & release pipeline. Here we tests 12 packages, which means 12 pipelines in ADO. The minimum number of rootfs I can do is 3 after many experiments trying to combine them.

Reply:
> Can the docker images be compiled together? How long does this suite take to execute?

About 30min in total.

Reply:
Have you been able to try a multi-layer dockerfile? You can build one layer with one config, extract the binaries to another layer and so on, so reduce the pipeline add time it would possibly take. You could also publish this container to dockerhub and consume from there

Reply:
Yes, I thought about multi-stage build but it doesn't apply to python-azure-sdk. If you look at the Dockerfile, it doesn't build any binaries such that I can copy them to a smaller base image. Line 1 is the base image used by both pip install and pytest.

Reply:
> How long does it take to build all three rootfses? If taking too long, we can push the images to dockerhub like Saloni suggested.

It took 13:24min to build all three rootfs. Caching the docker image reduced the time to 5:24min. I will try to publish the image.

Reply:
Done